src/System.IO.FileSystem/tests/FileStream/CanSeek.Windows.cs

```diff
@@ -1,25 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using Microsoft.Win32.SafeHandles;
-using System;
-using System.IO;
-using Xunit;
-
-namespace System.IO.FileSystem.Tests
-{
-    partial class FileStream_CanSeek
```

Comment:
Yup, I made it work on both Windows and Unix, so I just moved it out of the Windows-only file.